test/01-constrain.test.ts

```diff
@@ -73,5 +76,41 @@ describe('01 basic constrain tests', function () {
 					'It is necessary that each student that has a semester credits, has a semester credits that is greater than or equal to 4 and is less than or equal to 16.',
 				);
 		});
+
```

Comment:
I know it's verbose, but please move this test to a new 0x-webresource.test.ts as it's testing a complete new finctionality.

This test file should contain all kinds of SBVR rules / constraints that can be defined. The WebResource is not a constrain as is but a more complex functionality